sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/repository/repository/ReactiveTeacherRepository.java

```diff
@@ -20,7 +20,7 @@ public interface ReactiveTeacherRepository extends ReactiveCosmosRepository<Reac
 
     Mono<Boolean> existsByLastNameIsNull();
 
-    @Query(value = "SELECT * FROM a WHERE ARRAY_CONTAINS(@firstNames, a.firstName) ")
+    @Query(value = "SELECT * FROM a WHERE ARRAY_CONTAINS(@firstNames, a.firstName) order by a.id ")
```

Comment:
why changing the query? should we add another test for the order by?

Reply:
There is already another one for order by :)
Changing this one so that we can match the results 100% of the time always, currently they come unordered and can come in any order without any guarantee, causing the test to be flaky.